features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapListener.java

```diff
@@ -102,15 +108,49 @@ public void trapError(int error, String msg) {
     }
 
     public void start() {
-        m_twinSubscription = m_twinSubscriber.subscribe(TrapListenerConfig.TWIN_KEY, TrapListenerConfig.class, (config) ->  {
+        if (m_twinSubscriber != null) {
+            subscribe();
+        } else {
+            new Timer().schedule(new TimerTask() {
```

Comment:
Could use lambda expression here

Reply:
TimerTask isn't actually a functional interface, so you can't use lamda functions to define them.